smrf/framework/art.py

```diff
@@ -0,0 +1,34 @@
+# flake8: noqa
+
+title1 = [
+    "  .----------------.  .----------------.  .----------------.  .----------------.",
+    " | .--------------. || .--------------. || .--------------. || .--------------. |",
+    " | |    _______   | || | ____    ____ | || |  _______     | || |  _________   | |",
+    " | |   /  ___  |  | || ||_   \  /   _|| || | |_   __ \    | || | |_   ___  |  | |",
+    " | |  |  (__ \_|  | || |  |   \/   |  | || |   | |__) |   | || |   | |_  \_|  | |",
+    " | |   '.___`-.   | || |  | |\  /| |  | || |   |  __ /    | || |   |  _|      | |",
+    " | |  |`\____) |  | || | _| |_\/_| |_ | || |  _| |  \ \_  | || |  _| |_       | |",
+    " | |  |_______.'  | || ||_____||_____|| || | |____| |___| | || | |_____|      | |",
+    " | |              | || |              | || |              | || |              | |",
+    " | '--------------' || '--------------' || '--------------' || '--------------' |",
+    "  '----------------'  '----------------'  '----------------'  '----------------' ",
+    " "]
+
+title2 = [
+    "    SSSSSSSSSSSSSSS  MMMMMMMM               MMMMMMMM RRRRRRRRRRRRRRRRR    FFFFFFFFFFFFFFFFFFFFFF",
+    "  SS:::::::::::::::S M:::::::M             M:::::::M R::::::::::::::::R   F::::::::::::::::::::F",
+    " S:::::SSSSSS::::::S M::::::::M           M::::::::M R::::::RRRRRR:::::R  F::::::::::::::::::::F",
+    " S:::::S     SSSSSSS M:::::::::M         M:::::::::M RR:::::R     R:::::R FF::::::FFFFFFFFF::::F",
+    " S:::::S             M::::::::::M       M::::::::::M   R::::R     R:::::R   F:::::F       FFFFFF",
+    " S:::::S             M:::::::::::M     M:::::::::::M   R::::R     R:::::R   F:::::F",
+    "  S::::SSSS          M:::::::M::::M   M::::M:::::::M   R::::RRRRRR:::::R    F::::::FFFFFFFFFF",
+    "   SS::::::SSSSS     M::::::M M::::M M::::M M::::::M   R:::::::::::::RR     F:::::::::::::::F",
+    "     SSS::::::::SS   M::::::M  M::::M::::M  M::::::M   R::::RRRRRR:::::R    F:::::::::::::::F",
+    "        SSSSSS::::S  M::::::M   M:::::::M   M::::::M   R::::R     R:::::R   F::::::FFFFFFFFFF",
+    "             S:::::S M::::::M    M:::::M    M::::::M   R::::R     R:::::R   F:::::F",
+    "             S:::::S M::::::M     MMMMM     M::::::M   R::::R     R:::::R   F:::::F",
+    " SSSSSSS     S:::::S M::::::M               M::::::M RR:::::R     R:::::R FF:::::::FF",
+    " S::::::SSSSSS:::::S M::::::M               M::::::M R::::::R     R:::::R F::::::::FF",
+    " S:::::::::::::::SS  M::::::M               M::::::M R::::::R     R:::::R F::::::::FF",
+    "  SSSSSSSSSSSSSSS    MMMMMMMM               MMMMMMMM RRRRRRRR     RRRRRRR FFFFFFFFFFF",
+    " "]
```

Comment:
Pick your favorite! :-)

Reply:
There used to be a config option, not sure what happened to it!